examples/helloworld/helloworld.py

```diff
@@ -0,0 +1,95 @@
+import botocore
```

Comment:
Not blocking, but there should be a test command for examples (I'll create an issue for it separately).